Directory.Build.targets

```diff
@@ -15,6 +15,7 @@
     <!-- Do not upgrade this version or we won't support old SDK -->
     <PackageReference Update="Newtonsoft.Json" Version="9.0.1" />
     <PackageReference Update="NuGet.Packaging" Version="5.4.0" />
+    <PackageReference Update="NuGet.Frameworks" Version="5.11.0" />
```

Comment:
is this compat with the SDK one right? https://github.com/dotnet/sdk

Reply:
I think it was a conflict with `vstest` and the objectmodel it's using. https://github.com/microsoft/vstest/blob/main/scripts/build/TestPlatform.Dependencies.props#L57

Reply:
But I don't know how to align our version with theirs. So that `coverlet` doesn't break when they change their version.